src/components/KSelect/KSelectItem.vue

```diff
@@ -1,18 +1,20 @@
 <template>
   <div
     :key="item.key"
-    :aria-selected="item.selected"
     class="select-item"
     :data-testid="`select-item-${item.value}`"
-    role="option"
```

Comment:
I believe this is the case of a bad merge? @portikM didn't you move this role attribute, etc. to this parent?

Reply:
let me fix the bad rebase; for my changes I just need tabindex="0" **change** for this file.

Reply:
fixed.